test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1149,3 +1181,105 @@ UCS_TEST_P(test_uct_sockaddr_multiple_cms, server_switch_cm)
 }
 
 UCT_INSTANTIATE_SOCKADDR_TEST_CASE(test_uct_sockaddr_multiple_cms)
+
+class test_uct_sockaddr_legacy : public test_uct_sockaddr
```

Comment:
this class is to test the previous flow only?
why do we need it if the new code covers both cases.

Reply:
> this class is to test the previous flow only?

yes
> why do we need it if the new code covers both cases.

this is implementation details, need to cover API as well

Reply:
pls add comment for this test class